dags/catalyst.py

```diff
@@ -0,0 +1,46 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [jetstream repository](https://github.com/mozilla/catalyst).
```

Comment:
```suggestion
See the [catalyst repository](https://github.com/mozilla/catalyst).
```

Reply:
Thanks, not sure how I missed all of these!